server/db_setup.sql

```diff
@@ -10,7 +10,8 @@ create table laptop_request (
     lastname varchar(100),
     email varchar(100),
     phonenumber varchar(100),
-    laptop_request_status laptop_request_status DEFAUlT 'ACTIVE'
+    laptop_request_status laptop_request_status DEFAUlT 'ACTIVE',
+    uuid text
```

Comment:
It's probably a good idea to set a UNIQUE index on this uuid field (and the one in the next table) so that the database knows it's unique and can then retrieve rows more efficiently (given that queries are searching on this field). While it won't make much difference when the database is small, these constraints can help when a database gets bigger.

You probably also want to declare a NOT NULL constraint on the field, because this field should always be set.

You can see examples here: https://www.postgresql.org/docs/14/ddl-constraints.html

Reply:
Fixed that